validation/longhorn/schemas/pit_schemas.md

```diff
@@ -0,0 +1,399 @@
+| 6 | Navigate to "Node" section in Longhorn UI | | Node management page loads |
+| 7 | Navigate to "Setting" section in Longhorn UI | | Settings page loads |
+| 8 | Return to Rancher tab and verify session maintained | | Rancher session still active |
+| 9 | Create test volume from Longhorn UI | | Volume creation succeeds |
+| 10 | Return to Rancher and verify volume appears | | Volume visible in Rancher Longhorn section |
+
+### RBAC Integration Testing
+
+Test Role-Based Access Control integration between Rancher and Longhorn for different user permission levels.
+
+| Step Number | Action | Data | Expected Result |
+|------|--------|--------|-----------------|
+| 1 | Create cluster admin user in Rancher | | User created with cluster-admin role |
+| 2 | Create project member user in Rancher |  | User created with project-member role |
+| 3 | Create read-only user in Rancher | | User created with read-only permissions |
+| 4 | Login as cluster admin and access Longhorn UI | | Full access to all Longhorn features |
+| 5 | Test volume creation and deletion as cluster admin | | Operations succeed without restrictions |
+| 6 | Login as project member and access Longhorn UI | | Access granted with project scope |
+| 7 | Test volume operations within project scope | | Operations succeed within assigned project |
+| 8 | Login as read-only user and access Longhorn UI | | Read-only access granted |
+| 9 | Attempt volume creation as read-only user | | Operation blocked with appropriate error message |
+| 10 | Verify RBAC consistency across both UIs | | Permissions enforced consistently |
+
+### Volume Creation Through Rancher Workloads
+
+Verify automatic volume provisioning and attachment when deploying workloads with Longhorn storage through Rancher.
+
+| Step Number | Action | Data | Expected Result |
+|------|--------|--------|-----------------|
+| 1 | Navigate to "Workloads" in Rancher cluster view | | Workloads section displays |
+| 2 | Click "Create" to deploy new workload | | Workload creation form opens |
+| 3 | Configure workload with name `test-longhorn-workload` and nginx image | | Basic workload configuration set |
+| 4 | Click "Add Volume" → "Add a new persistent volume (claim)" | | Volume configuration dialog opens |
+| 5 | Configure PVC: name=`test-pvc`, storageClass=`longhorn`, size=`1Gi`, accessMode=`ReadWriteOnce` | | PVC settings configured |
+| 6 | Set mount path to `/data` | | Mount path configured |
+| 7 | Deploy the workload | | Workload deployment begins |
+| 8 | Verify workload pod reaches Running state | | Pod shows "Running" status |
+| 9 | Check Longhorn UI for automatic volume creation | | Volume appears in Longhorn UI as "Attached" |
+| 10 | Exec into pod and write test data to `/data` | | Data write succeeds |
+| 11 | Delete and recreate pod, verify data persistence | | Data persists across pod restart |
+
+### StatefulSet Volume Management
```

Comment:
would reword to something that can reflect the scaling nature of the test. maybe `Create and Scale StatefulSet with PVC Template`

Reply:
Done